components/chat/chatBar/CatalogListBox.tsx

```diff
@@ -0,0 +1,307 @@
+                  placement="right"
+                  isOpen={focusedItem === tool.index}
+                  onOpenChange={(open) =>
+                    setFocusedItem(open ? tool.index : null)
+                  }
+                  closeDelay={0.5}
+                  classNames={{
+                    content: 'max-w-[250px]',
+                  }}
+                >
+                  {tool.name}
+                </Tooltip>
+              </ListboxItem>
+            ))}
+          </ListboxSection>
+        ))}
+      </Listbox>
+    );
+  }
+);
+
+function focusElement(id: string) {
+  const element = document.getElementById(id);
+  if (!element) return;
+  element.focus();
+}
+
+// note(tylerslaton) - This will eventually be retrieved from the tools site, for now we must endure the pain of hardcoding.
+const featuredTools: FeaturedTools = {
+  'AI and Web': [
+    {
+      name: 'Vision',
+      description: 'Allows the assistant to interact with images.',
+      url: 'github.com/gptscript-ai/gpt4-v-vision@gateway',
+      tags: ['vision', 'images', 'ai'],
+      icon: <BsEye className="text-lg" />,
+    },
+    {
+      name: 'Image Generation',
+      description: 'Allows the assistant to generate images.',
+      url: 'github.com/gptscript-ai/dalle-image-generation@gateway',
+      tags: ['images', 'ai', 'generation'],
+      icon: <FaPaintBrush className="text-lg" />,
+    },
+    {
+      name: 'Search The Internet',
+      description: 'Allows the assistant to search the web for answers.',
+      url: 'github.com/gptscript-ai/answers-from-the-internet',
+      tags: ['search', 'web', 'internet'],
+      icon: <GoGlobe className="text-lg" />,
+    },
+    {
+      name: 'Crawl Website',
+      description: 'Allows the assistant to search a website.',
+      url: 'github.com/gptscript-ai/search-website',
+      tags: ['search', 'web', 'site'],
+      icon: <GoSearch className="text-lg" />,
+    },
+    {
+      name: 'Browser',
+      description:
+        'Provides the assistant with the ability to interact with the web via a Chrome window.',
+      url: 'github.com/gptscript-ai/browser',
+      tags: ['browser', 'web', 'chrome', 'search'],
+      icon: <GoBrowser className="text-lg" />,
+    },
+  ],
+  Productivity: [
+    {
+      name: 'Slack',
+      description: 'Allows the assistant to interact with Slack.',
+      url: 'github.com/gptscript-ai/tools/apis/slack/write',
+      tags: ['slack', 'messaging', 'teams', 'api'],
+      icon: <AiOutlineSlack className="text-lg" />,
+    },
+    {
+      name: 'Notion',
+      description: 'Allows the assistant to interact with Notion.',
+      url: 'github.com/gptscript-ai/tools/apis/notion/write',
+      tags: ['notion', 'documentation', 'notes', 'api'],
+      icon: <SiNotion className="text-lg" />,
+    },
+    {
+      name: 'Hubspot',
+      description: 'Allows the assistant to interact with Hubspot.',
+      url: 'github.com/gptscript-ai/tools/apis/hubspot/crm',
```

Comment:
i have already fixed this, but just fyi the split has happened and this is no longer the correct url. its now `github.com/gptscript-ai/tools/apis/hubspot/crm/write`